drivers/scheduler/k8s/k8s.go

```diff
@@ -827,6 +827,7 @@ func (k *K8s) Schedule(instanceID string, options scheduler.ScheduleOptions) ([]
 				Enabled:  app.Enabled,
 			},
 			ScheduleOptions: options,
+			SkipMountOptionValidation: true,
```

Comment:
Instead of having `SkipMountOptionValidation`, we can derive this from two other places:
* We have an `IS_PURE_VOLUMES` variable we set for all DA tests: this will let us filter to only run the check during Pure tests. [Here is an example using it.](https://github.com/portworx/torpedo/blob/master/drivers/scheduler/k8s/k8s.go#L715)
* We already have to fetch the storage class to check if the mount options contain `nosuid`: we can use this to filter to run the test only when it matters